Index: Makefile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Makefile b/Makefile
--- a/Makefile	(revision bcea3c7ef01dac6d7f9f49ebf9e90fe66d86f5f7)
+++ b/Makefile	(date 1708375093373)
@@ -51,7 +51,7 @@
 		CCX     := clang++
 		LD      := ld.lld
 		CDIR    := linux
-		AR      := llvm-ar
+		AR      := ar
 endif
 ifeq ($(UNAME_S),Darwin)
 		CC      := /usr/local/opt/llvm/bin/clang
@@ -66,7 +66,7 @@
 	$(TOOLCHAIN)/bin/$(CDIR)/create-fself -in=$(INTDIR)/$(PROJDIR).elf -out=$(INTDIR)/$(PROJDIR).oelf --lib=$(TARGET) --paid 0x3800000000000011

 $(TARGETSTATIC): $(INTDIR) $(OBJS)
-	$(AR) --format=bsd rcs $(TARGETSTATIC) $(TARGETCRT) $(INTDIR)/*.o
+	$(AR) rcs $(TARGETSTATIC) $(TARGETCRT) $(INTDIR)/*.o

 $(TARGETSTUB): $(INTDIR) $(STUBOBJS)
 	$(CC) $(INTDIR)/*.o.stub -o $(TARGETSTUB) -target x86_64-pc-linux-gnu -shared -fuse-ld=lld -ffreestanding -nostdlib -fno-builtin -L$(TOOLCHAIN)/lib $(LIBS)
